cdp_scrapers/legistar_utils.py

```diff
@@ -488,6 +488,9 @@ def get_legistar_content_uris(client: str, legistar_ev: dict) -> ContentUriScrap
         # that is a summary-like page for a meeting
         with urlopen(legistar_ev[LEGISTAR_EV_SITE_URL]) as resp:
             soup = BeautifulSoup(resp.read(), "html.parser")
+            if "server error" in soup.text.lower():
```

Comment:
We may consider making this parametrized. For now this is the first case of something like this; HTTP 200 returned but is in fact not accessible. If and when we encounter more cases, we can revisit.

Reply:
Oh actually I thought it was good that it started error -- it alerts us it is down. Just that it wasn't clear that is what was happening. Can you raise this as an error but with the context of "Legistar appears to be down" or something similar.

Reply:
That's a good point. 💡 Agreed.